core/arch/arm/include/sm/optee_smc.h

```diff
@@ -422,6 +422,32 @@
 #define OPTEE_SMC_VM_DESTROYED \
 	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_VM_DESTROYED)
 
+/*
+ * Query OP-TEE about number of supported threads
+ *
+ * Normal World OS or Hypervisor issues this call to find out how much threads
+ * OP-TEE support. In other hand, how many standard calls can be issued in
+ * parallel before OP-TEE will return OPTEE_SMC_RETURN_ETHREAD_LIMIT.
+ *
+ * Call requests usage:
+ * a0	SMC Function ID, OPTEE_SMC_GET_THREAD_COUNT
+ * a1-6 Not used
+ * a7	Hypervisor Client ID register
+ *
+ * Normal return register usage:
+ * a0	OPTEE_SMC_RETURN_OK
+ * a1   Number of threads
+ * a2-6 Requested information
```

Comment:
What is returned in a2-a6?

Reply:
It's the leftover from the previous version. Good catch, thank you.